### ISSUE TITLE
Edges are drawn in front of vertices with Plotly backend

### ISSUE THREAD

**Issue by szhorvat** [igraph/python-igraph#691]
**Describe the bug**

Edges are drawn in front of vertices with Plotly backend. There is a visible overlap.

**To reproduce**

```python
import igraph as ig
ig.plot(ig.Graph.Ring(10), vertex_size=100, backend="plotly")
```

<img width="982" alt="image" src="https://github.com/igraph/python-igraph/assets/1212871/a7dcb4be-b7cc-4675-8c1f-14b37ccc68aa">

**Version information**

Current `main`.

**Comment by ntamas** [comment]
I've tried to tackle this but a simple reordering of the drawing calls in the Plotly graph drawer did not seem to do the trick and I'm not familar with the internals of Plotly enough.

**Comment by iosonofabio** [comment]
Happy to take this, just came back from holiday

**Comment by iosonofabio** [comment]
I'll take a look on top of `develop` ASAP

**Comment by meghanmccreary** [comment]
If it helps anyone else - adding `layer="below"` to the add_shape calls in drawing/plotly/edge.py worked for me:
`fig.add_shape(stroke, layer="below")`

Not necessarily a robust fix with z-order flow through and all that, but it was good enough for my purpose.

**Comment by ntamas** [comment]
Let me test this real quick and if it works, I'll commit it as a hotfix before we release 0.11.0.

**Comment by ntamas** [comment]
Fixed in fe6302cc
